### PR TITLE
[RHPAM-3596] - Added a managed dependency for jsr305

### PR DESCRIPTION
**Pull request by akoufoudakis**
**Thank you for submitting this pull request**

**JIRA**: RHPAM-3596 

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest</b> (using <i>this</i> e.g. <b>Jenkins retest this</b> optional but no longer required)
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by akoufoudakis** [comment]
Jenkins run fdb

**Comment by akoufoudakis** [comment]
Jenkins run fdb

**Comment by mareknovotny** [comment]
i think this change is going backward with the current status on master branch.
We already have guava on 30.1-jre version which has different findbugs/jsr305 dependency version and enforcing old problematic one is a no go.

File: pom.xml [hunk 1]
```diff
@@ -2459,6 +2460,12 @@
         <version>${version.com.google.protobuf}</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.google.code.findbugs</groupId>
+        <artifactId>jsr305</artifactId>
+        <version>${version.com.google.code.findbugs.jsr305}</version>
+      </dependency>
+
```
Comment (mareknovotny) [inline pom.xml:2468]: What version is used without this depMgmt declaration?

**Comment by akoufoudakis** [comment]
Master is covered by guava:30.1

**Pull request closed without merging**